CPS-???/README.md

```diff
@@ -0,0 +1,34 @@
+---
+CPS: ?
+Title: Burning NFTs / tokens
+Status: Open
+Category: Tokens
+Authors:
+    - Mateusz Czeladka <mateusz.czeladka@cardanofoundation.org>
+Proposed Solutions: 
+    - smart contract / native script that is able to burn token
+
+Created: 2022-12-01
+---
+
+## Abstract
+
+The aim of this problem statement is to raise the issue around the ability to burn NFT / tokens and release minUTxO (ADA) backing the NFT / token.
+
+## Problem
+
+It is understood that minUTxO has been introduced as a way to protect ledger from tokens and NFTs spam, however, the only currently supported methods of "burning NFTs" do not address the core of the problem, namely releasing ADA 'locked' behind a token or NFT. Currently recommended methods of 'burning' are either sending to another wallet owned by the person sending or send to another wallet owned by somebody else. Neither of those methods actually releases underlying 'locked' minUTxO ADA.
```

Comment:
Thank you for writing this CPS up. It might also be worth mentioning that since the deposit scaling is affine, there is some incentive to keep bundling the tokens into bigger and bigger transaction outputs, potentially making the situation worse. It's unclear what the expectation should be regarding accepting payments in ADA that may be mired in tokens.